rocket-chat-android-widgets/src/main/res/values/message_styles.xml

```diff
@@ -17,6 +17,9 @@
 
     <style name="TextAppearance.RocketChat.MessageAttachment.Title" parent="TextAppearance.AppCompat.Title">
         <item name="android:textSize">14sp</item>
+    </style>
+
+    <style name="TextAppearance.RocketChat.MessageAttachment.Title.Link" parent="TextAppearance.RocketChat.MessageAttachment.Title">
```

Comment:
Also, I think it is better not to define this and simply use textAppearance=`@style/TextAppearance.RocketChat.MessageAttachment.Title` and `android:textColor="?android:attr/textColorLink"` manually.

If this style will be enhanced in future (ex. bold, big textSize...), the name "Link" would be invalid and another name such as "MessageAttachment.Title.Decorated" or "MessageAttachment.Title.Emphasized" would be nice, I think.